.github/workflows/scraper.yml

```
@@ -49,3 +49,10 @@ jobs:
         else
             echo "There were no changes. Nothing to do."
         fi
+    - name: Upload screenshots
+      if: failure()
+      uses: actions/upload-artifact@v4
+      with:
+        name: scraper_screenshot
+        path: ./tmp/capybara
+        retention-days: 2
```

```suggestion
        retention-days: 5
```

pongámosle unos días más por si falla un fin de semana por ejemplo y no lo vemos enseguida